charts/argo-cd/templates/redis/deployment.yaml

```diff
@@ -0,0 +1,61 @@
+{{- if .Values.redis.enabled }}
```

Comment:
@alexec do you think redis can be disabled? I know it is used like a throw away cache, but I also think that argo-cd expects it to be there and I am not sure it will function correctly if it doesn't exist.

Reply:
Eventually I think bot Dex and Redis should be moved to sub charts.  I don't see a huge value in managing the charts for these two apps as they both have very well documented and supported charts of their own.

Reply:
Good question - we're looking into writing an operator that install Argo CD in a "cluster only" mode. That would not include Redis. I think it is therefore fine - going forward - to disable it.

Reply:
I defaulted redis.enabled to false: https://github.com/argoproj/argo-helm/pull/129/commits/37e0797a1d59476121f686db81f42d91d18e7265